components/comments/templates/post.html

```diff
@@ -88,7 +88,7 @@
 
 					<div class="leaveComment" placeholder="<%- placeholder || dplace %>"></div>
 					
-					<% if(embedimage) {%>
+					<% if(true) {%>
```

Comment:
Why not just removing this conditional construction @MoMentalochka?

Reply:
miss play detected

Reply:
Fore reference: this conditional construction was changed for tests only. Changes are made, can proceed with this...